posts/api_enabled_retail.md

```diff
@@ -0,0 +1,67 @@
+# Modern APIs are Pushing Innovation in Retail
+
+An increasing number of retailers are API enabling core parts of their business, delivering a more cohesive customer experience across a variety of touchpoints, making it easier to streamline internal operations, and opening opportunities to interface with external channels.
```

Comment:
"API enabling" is a weird phrase IMO. "exposing core parts of their business through APIs"?